src/libertem/web/messages.py

```diff
@@ -7,7 +6,7 @@ class Message(object):
     possible messages - the translation of our python datatypes to json types
     """
 
-    def __init__(self, state: SharedState):
+    def __init__(self, state):
         self.state = state
```

Comment:
I did this to get rid of a circular import issue. If you really want the SharedState just for typing, the whole server "base" file will need some rethinking/restructure. I guess that's better for you to handle as you're more experienced with the code base.